src/WebJobs.Script/Diagnostics/Sanitizer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Azure.WebJobs.Logging
     // Note: This file is shared between the WebJobs SDK and Script repos. Update both if changes are needed.
     internal static class Sanitizer
     {
-        private const string SecretReplacement = "[Hidden Credential]";
+        public const string SecretReplacement = "[Hidden Credential]";
```

Comment:
Below, I wonder if while we're in here we shouldn't apply the change made in https://github.com/Azure/azure-functions-host/commit/450b886fe5822c10ffea5c7a6c962cd8f43d2cff to CredentialTokens?